tutor_webhook_receiver/plugin.py

```diff
@@ -2,14 +2,44 @@
 import os
 import pkg_resources
 
+from .__about__ import __version__
 
 templates = pkg_resources.resource_filename(
     "tutor_webhook_receiver", "templates"
 )
 
-config = {}
+config = {
+    "defaults": {
+        "VERSION": __version__,
+        "DOCKER_IMAGE": "webhook_receiver/webhook_receiver:latest",
+        "HOST": "webhooks.{{ LMS_HOST }}",
+        "DB_ENGINE": "django.db.backends.mysql",
+        "DB_NAME": "webhook_receiver",
+        "DB_USER": "webhook_receiver01",
+        "DB_PASSWORD": "",
+        "DB_HOST": "{{ MYSQL_HOST }}",
+        "DB_PORT": "{{ MYSQL_PORT }}",
+        "DB_MIGRATION_OPTIONS": {},
+        "DJANGO_SECRET_KEY": "",
+        "DJANGO_LOG_LEVEL": "DEBUG",
+        "LMS_BASE_URL": "",
+        "EDX_OAUTH2_KEY": "",
```

Comment:
I am *guessing* Tutor has hooks for setting these up, too. After all they are needed for Studio and MFEs.

Reply:
Yes I think I'll just add a hook for this as in the ecommerce plugin.